platform/broadcom/sonic-platform-modules-cel/haliburton/script/cpu_wdt

```diff
@@ -79,6 +79,7 @@ if [[ "${ACTION}" == "start" ]]; then
     log_info "Enable keep alive messaging every $KEEPALIVE seconds"
     while [[ ${CPUWDT_MAIN_TASK_RUNNING_FLAG} == "true" ]]; do
         watchdogutil arm -s "${TIMEOUT}" > /dev/null
```

Comment:
@lizhijianrd do we need to redirect the log to /dev/null ?. I think its OK to not redirect to /dev/null